guides/hosting/installation-updates/deployments/build-w-o-db.md

```diff
@@ -71,3 +27,7 @@ storefront:
  ```
 
  This will force the theme compiler to use the static dumped file instead of looking into the database.
+
+### Partially compiling the Storefront
+
+You can also build just the Javascript bundle using `CI=1 SHOPWARE_SKIP_THEME_COMPILE=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true bin/build-storefront.sh'` (without the need for the above loader) in your CI and later run `bin/console theme:dump` on your production system, when the database is available. This will happen anyways, if theme variables are changed via the admin panel.
```

Comment:
```suggestion
You can also build just the Javascript bundle using `CI=1 SHOPWARE_SKIP_THEME_COMPILE=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true bin/build-storefront.sh` (without the need for the above loader) in your CI. After that, run `bin/console theme:dump` on your production system, when the database is available. This will happen automatically, if theme variables are changed via the admin panel.
```